python/lsst/pipe/tasks/postprocess.py

```diff
@@ -254,7 +292,7 @@ class PostprocessAnalysis(object):
 
 
     """
-    _defaultRefFlags = ('calib_psf_used', 'detect_isPrimary')
```

Comment:
@timothydmorton : Why is 'calib_psf_used' in the `_defaultFlags` list?  It doesn't appear in every catalog.  Is there someplace downstream in qa_explorer for which you were depending on this that I can update?

Reply:
It seemed like something we were looking at a lot in QA, so I put it in here; I don't really know where it does or doesn't appear.  No reason this couldn't be changed.